content/500-support/100-help-articles/300-finding-entities-based-on-relation.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: 'Finding entities based on their relation existence'
+metaTitle: 'Finding entities based on their relation existence'
```

Comment:
```suggestion
title: 'Querying models based on their relations'
metaTitle: 'Querying models based on their relations'
```

Reply:
@nikolasburk I think **Querying models based on their relations** seems too generic and could be anything but this article is specific should it be something like ****Querying models based on their relations existence**